### ISSUE TITLE
InlineDocCommentDeclaration false positive

### ISSUE THREAD

**Issue by dereuromark** [slevomat/coding-standard#1297]
### Bug report

I wanted to add 
```xml
<rule ref="SlevomatCodingStandard.Commenting.InlineDocCommentDeclaration" />
```
for us.

But inside a method:
```php
/** @var \PHPStan\PhpDocParser\Ast\PhpDoc\MethodTagValueNode|\PHPStan\PhpDocParser\Ast\PhpDoc\PropertyTagValueNode|\PHPStan\PhpDocParser\Ast\PhpDoc\VarTagValueNode|\PHPStan\PhpDocParser\Ast\PhpDoc\ParamTagValueNode|\PHPStan\PhpDocParser\Ast\PhpDoc\ReturnTagValueNode|\PHPStan\PhpDocParser\Ast\PhpDoc\GenericTagValueNode $valueNode */
if ($valueNode instanceof MethodTagValueNode) {
```        
reports
```
FOUND 1 ERROR AFFECTING 1 LINE
---------------------------------------------------------------------------------------------------------------------------------------------------------------------
 59 | ERROR | Missing variable $valueNode before or after the documentation comment.
    |       | (SlevomatCodingStandard.Commenting.InlineDocCommentDeclaration.MissingVariable)
```

But it is there and correct, I cannot find a reason for this error to appear. Also no additional whitespace etc.
PHPStan likes it.

**Comment by dereuromark** [comment]
I see, it thinks that the assignment itself should be the one declaring it.
Well, in that case it comes from a method, which signature we cannot change.

But in some cases maybe the docblock could be adjusted, or as in this case, I can just use `phpstan-var` tag, since this is only to make that tool happy.
So I guess, this is working then as it should.

**Comment by dereuromark** [comment]
Well, no, the lenght of the line must be the issue, it really seems to be a bug of some sorts.
